doc/source/examples/multiphysics/rayleigh-benard-convection/rayleigh-benard-convection.rst

```diff
@@ -0,0 +1,181 @@
+        set output time                     = 25
+        set output path                     = ./output/     
+    end
+
+
+The ``multiphysics`` subsection enables to turn on `(true)` and off `(false)` the physics of interest. Here ``heat transfer``, ``buoyancy force``, and ``fluid dynamics`` are chosen.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Multiphysics
+    #---------------------------------------------------
+    subsection multiphysics
+        set buoyancy force      = true
+        set heat transfer       = true
+        set fluid dynamics      = true
+    end 
+    
+The ``source term`` subsection defines gravitational acceleration. The value of the gravitational acceleration in this example is selected to satisfy the desired values of Ra number.
+
+.. code-block:: text
+    
+    #---------------------------------------------------
+    # Source term
+    #---------------------------------------------------
+    subsection source term
+        set enable                      = true
+        subsection xyz
+            set Function expression     = 0 ; -10 ; 0
+        end
+    end
+
+
+The ``physical properties`` subsection defines the physical properties of the fluid. Since we simulate the Rayleigh-Benard convection at two Rayleigh numbers (Ra=10000 and 25000), we use different thermal conductivities to reach mentioned Rayleigh numbers. We change the thermal conductivity of the fluid in the two simulations. Note that any other physical property (that is present in the Rayleigh number equation defined above) can be used instead of thermal conductivity. Both thermal conductivity values (:math:`k=0.15625` for :math:`Ra=10000`, and :math:`k=0.15625` for :math:`Ra=25000`) are added to the parameter handler file. However, only one of them should be uncommented for each simulation.
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Physical Properties
+    #---------------------------------------------------
+    subsection physical properties
+        set number of fluids            = 1
+        subsection fluid 0
+            set density                 = 100
+            set kinematic viscosity     = 0.01
+            set thermal expansion       = 0.0002
+            set thermal conductivity    = 0.15625	# for Ra = 10000
+            #set thermal conductivity   = 0.0625	# for Ra = 25000
+            set specific heat           = 100
+        end
+    end
+
+---------------------------
+Running the simulation
+---------------------------
+
+Call the gls_navier_stokes_2d by invoking:  
+
+``mpirun -np 8 gls_navier_stokes_2d rayleigh_benard_convection.prm``
+
+to run the simulation using eight CPU cores. Feel free to use more.
+
+
+.. warning:: 
+    Make sure to compile lethe in `Release` mode and 
+    run in parallel using mpirun. This simulation takes
+    :math:`\approx` 20 minutes on 8 processes.
+
+
+-------
+Results
+-------
+
```

Comment:
Is there an experimental/analytical result that we can use for comparison?

Reply:
Good question, that would be very relevant!

Reply:
The only analysis that I could perform was the comparison of the number of vortices at different Rayleigh numbers. The behavior of flow in Rayleigh-Bénard convection is extremely unstable which causes bifurcation of the flow. This makes it very hard to compare other parameters with other experiments/simulations.